lib/cocoapods/sources_manager.rb

```diff
@@ -649,5 +124,33 @@ def update_git_repo(show_output = false)
       end
       super
     end
+
+    def verify_compatibility!
+      super
+      latest_cocoapods_version = metadata.latest_cocoapods_version && Gem::Version.create(metadata.latest_cocoapods_version)
+      return unless Config.instance.new_version_message? &&
+        latest_cocoapods_version &&
+        latest_cocoapods_version > Gem::Version.new(Pod::VERSION)
+
+      rc = latest_cocoapods_version.prerelease?
+      install_message = !Pathname(__FILE__).dirname.writable? ? 'sudo ' : ''
+      install_message << 'gem install cocoapods'
+      install_message << ' --pre' if rc
+      message = [
+        '',
+        "CocoaPods #{latest_cocoapods_version} is available.".green,
+        "To update use: `#{install_message}`".green,
+        ("[!] This is a test version we'd love you to try.".yellow if rc),
+        ("Until we reach version 1.0 the features of CocoaPods can and will change.\n" \
```

Comment:
We should probably remove this, because once it is released it'll be 1.0, right? :)